miniapp/io.cpp

```diff
@@ -389,6 +409,64 @@ std::ostream& operator<<(std::ostream& o, const cl_options& options) {
     return o;
 }
 
+
+/// Parse spike times from a stream
+/// A single spike per line, white space is ignored
+/// All characters after a parsed floating point nr on a line are ignored
+/// Throws a usage error when parsing fails
+///
+/// Returns a unique_ptr to a vector of time_type
+
+std::unique_ptr<std::vector<time_type> > parse_spike_times_from_stream(
+    std::ifstream & fid) {
```

Comment:
Done, although,

 util::sprintf does not exist, I used util::strprintf. 
I observed that this function eats the first character of the supplied string (to print the ""??). Is this expected behavior?